pkg/authorizations/authorizations.go

```diff
@@ -0,0 +1,52 @@
+package authorizations
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authoritytypes "github.com/zeta-chain/zetacore/x/authority/types"
+	crosschainTypes "github.com/zeta-chain/zetacore/x/crosschain/types"
+	fungibletypes "github.com/zeta-chain/zetacore/x/fungible/types"
+	lightclienttypes "github.com/zeta-chain/zetacore/x/lightclient/types"
+	observertypes "github.com/zeta-chain/zetacore/x/observer/types"
+)
+
+func AuthorizationTable() map[string]authoritytypes.PolicyType {
```

Comment:
Why using a function instead of defining a global var directly?

Reply:
There is no specific reason. I guess both options work.
The initial version of this pr , had some logic to deal with messages which need more than one policy type (before the refactor for light client pr was merged ) and thus needed a function . But it can be changed to a var now if needed.

Reply:
Var is easier to navigate with an IDE

Reply:
9485063a5dcc6ff1f35d38cf0c3f70a92507a9a5